common/display/displayplanemanager.cpp

```diff
@@ -228,8 +228,15 @@ bool DisplayPlaneManager::ValidateLayers(
               validate_final_layers = !(last_plane.GetOffScreenTarget());
 
             ResetPlaneTarget(last_plane, commit_planes.back());
+          } else {
+            // Plane separation is needed
+            if (!layer->IsSolidColor() && !layer->IsVideoLayer()) {
```

Comment:
>Why need this if judge?
The judgement is to avoid buffer allocation for planes intended not have overlay buffer attached to assigned plane. Here are the SolidColor layer and Video layer. 
>How about just combine ResetPlaneTarget(last_plane, commit_planes.back()); to line after line 222?
Tried this on debug environment, but it has some side effect. Though there is no black stripe issue but there are some rendering messy observed and effect is very similar to messy caused by RBC enabled scenario.

Reply:
OK thanks. Let's merge this patch first